### PR TITLE
Ruleset guide revise

### PR DESCRIPTION
**Pull request by Hainish**

### CODE REVIEW

File: ruleset-style.md [hunk 1]
```diff
@@ -21,8 +21,14 @@ which subdomains are covered.
...
 left-wildcard, along with a plain rewrite from "^http:" to "^https:". Make sure
 to add a bunch of test URLs for the more important subdomains. If you're not
-sure what subdomains might exist, check the 'subdomain' tab on Wolfram Alpha:
-https://www.wolframalpha.com/input/?i=_YOUR_DOMAIN_GOES_HERE_.
+sure what subdomains might exist, you can iterively use google queries and enumerate
+the list of results like such:
+
+1. site:*.eff.org
+2. site:*.eff.org -site:www.eff.org
+3. site:*.eff.org -site:www.eff.org -site:https://ssd.eff.org
```
Comment (jsha) [inline ruleset-style.md]: Why the https on the second one here?

File: ruleset-style.md [hunk 2]
```diff
@@ -21,8 +21,14 @@ which subdomains are covered.
 If you know all subdomains of a given domain support HTTPS, go ahead and use a
 left-wildcard, along with a plain rewrite from "^http:" to "^https:". Make sure
 to add a bunch of test URLs for the more important subdomains. If you're not
-sure what subdomains might exist, check the 'subdomain' tab on Wolfram Alpha:
-https://www.wolframalpha.com/input/?i=_YOUR_DOMAIN_GOES_HERE_.
+sure what subdomains might exist, you can iterively use google queries and enumerate
```
Comment (jsha) [inline ruleset-style.md]: Iteratively.

File: ruleset-style.md [hunk 3]
```diff
@@ -69,7 +75,7 @@ Here is an example ruleset pre-style guidelines:
 
 ```
 <ruleset name="WHATWG.org">
-  <target host="whatwg.org" />
+  <target host='whatwg.org' />
```
Comment (jsha) [inline ruleset-style.md]: Why did you change the quotes here? It's inconsistent with the rest of the example. Actually, we may want to add an entry saying "Use double quotes rather than single quotes," just for consistent.

Comment (Hainish) [inline ruleset-style.md]: This is made to illustrate how a rule may look _before_ the styleguide changes are applied.

One of the styleguide changes are to consistently use double quotes.  This is meant to illustrate that change.

File: ruleset-style.md (hunk 1 above)
Comment (Hainish) [inline ruleset-style.md]: Don't know how that got there.  Probably just copy/pasted it from a url.  This query is still valid - it does remove all ssd.eff.org entries.  But for consistency, I'll remove it.

File: ruleset-style.md (hunk 2 above)
Comment (Hainish) [inline ruleset-style.md]: hah! whoops

File: ruleset-style.md (hunk 3 above)
Comment (jsha) [inline ruleset-style.md]: Aha, should have read the context! Sounds good.

**Comment by Hainish** [comment]
Okay, ready for review again @jsha
